src/bn_balances.erl

```diff
@@ -84,7 +84,7 @@ load_block(_Hash, Block, _Sync, Ledger, State = #state{
                         fun({Entries, Type}) ->
                             lists:foldl(
                                 fun({AddressBin, EntryBin}, Acc) ->
-                                    HeightBin = integer_to_binary(Height),
+                                    HeightBin = binary:encode_unsigned(Height),
```

Comment:
While this will work we usually do not encode integers into binaries this way, since you are encoding the integer to a binary string form

```erlang
 HeightEntryKeyBin = <<AddressBin/binary, Height:64/integer-unsigned-little>>
```

is the way to encode an actual 64 bit binary value. We use little endian to store numbers in rocks

Reply:
This looks like it was resolved.